doc/whats-new.rst

```diff
@@ -21,6 +21,15 @@ v0.15.0 (unreleased)
 
 Breaking changes
 ~~~~~~~~~~~~~~~~
+- Bumped minimum tested versions for dependencies:
+  - numpy 1.15
+  - pandas 0.25
+  - scipy 1.3
+  - bottleneck 1.3
+  - h5py 2.10
+  - rasterio 1.1
```

Comment:
py36-min-all-deps.yml should have matching changes for bottleneck, h5py, rasterio

Reply:
Yeah I'm doing this incrementally since the env wasn't getting created locally.